src/extensions/bigbang/manifests.go

```diff
@@ -88,10 +88,22 @@ func manifestZarfCredentials(version string) corev1.Secret {
 
 // manifestGitRepo generates a GitRepository object for the Big Bang umbrella repo.
 func manifestGitRepo(cfg *extensions.BigBang) fluxSrcCtrl.GitRepository {
+	apiVersion := "source.toolkit.fluxcd.io/v1beta2"
+	semverVersion, err := semver.NewVersion(cfg.Version)
+	if err == nil {
+		c, err := semver.NewConstraint(">= 2.7.0")
+		if err == nil {
+			updateFlux, err := c.Validate(semverVersion)
+			if err == nil && updateFlux && !cfg.SkipFlux {
```

Comment:
Calling attention to this because I'm not sure what zarf wants to do here...there is the potential someone with `skipFlux` could have an older version of flux that doesn't support this newer v1 apiVersion so I added this check. Not sure if that's a good approach or not...I guess depends on zarf's posture towards skipping flux?

Reply:
so if the semantic version of cfg.Version is >= 2.7.0 then BB uses "source.toolkit.fluxcd.io/v1" else it uses "source.toolkit.fluxcd.io/v1beta2"?

Reply:
Yeah that's correct...as long as the BB extension config _does not_ skip flux. If the flux install is being skipped in the extension then this logic would leave it as `source.toolkit.fluxcd.io/v1beta2` given the unknown about the version they are using.